feature, do not launch plugins of missing required key. (#15)
autonity/autonity-oracle: plugins/README.md

# To write a new plugin
The oracle server provides a plugin framework for the development and management of data adaptors for loading data from external data source providers. Runtime management of plugins is dynamic; the server discovers and loads new or updated data source adaptors during runtime, so there is no need to restart the oracle server to detect data adaptor changes.

## The plugin configuration

> Every plugin has a unified configuration, it is configured by oracle server in the plugins-conf.yml file. When you implement a new plugin, you will need to prepare the default values for the plugin, for example:

Reviewer: ```suggestion
Every plugin has a unified configuration, that is configured by oracle server in the `plugins-conf.yml` file. When you implement a new plugin, you will need to prepare the default values for the plugin, for example:
```

// PluginState is the returned data when the oracle host want to initialise the plugin with basic information: version,
// and available symbols that the data source support.
type PluginState struct {
	KeyRequired      bool
	Version          string
	AvailableSymbols []string
}
```
where the statements in PluginState:

> - KeyRequired states that if the plugin need a service key to be configured, for a key required plugin, oracle server will not start the plugin if a key is missing from configuration.

Reviewer: ```suggestion
- KeyRequired states if the plugin needs a service key to be configured. When the plugin requires a key, oracle server will not start the plugin if a key is missing from the configuration.
```

- Version states the version of the plugin.

> - AvailableSymbols states the available symbols of current data plugin.

Reviewer: I find this hard to understand because of 'available'.  What does this mean?

Does it mean the set of symbols the oracle server is configured to fetch? 

I.e.:

```suggestion
- AvailableSymbols states the set of symbols the data plugin is configured to fetch.
```
